### PR TITLE
Warn if 2q repositories are in use

### PR DESCRIPTION
**Pull request by RichardPooleEDB**
After running most tpaexec commands, run the new playbook "check-repositories.yml", which does nothing but check the inventory for one host and warn if tpa_2q_repositories is set.

Fixes: TPA-631

### CODE REVIEW

**Comment by JonathanRenon-EDB** [comment]
Maybe we could add some mention on the decommission happening on Q? so that user have an idea on the time sensitivity of the matter ?

File: architectures/lib/commands/check-repositories.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+
+# © Copyright EnterpriseDB UK Limited 2015-2023 - All rights reserved.
+
+- name: Check for old repositories
+  hosts: all
+
+  tasks:
+  - assert:
+      that: "{{ tpa_2q_repositories is not defined }}"
```
Comment (amenonsen) [inline architectures/lib/commands/check-repositories.yml:10]: I think we should not warn if it's explicitly set to `[]`

Comment (RichardPooleEDB) [reply architectures/lib/commands/check-repositories.yml]: Done.

**Review by gonzalemario — APPROVED** [review]

File: architectures/lib/commands/check-repositories.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- name: Check for old repositories
+  hosts: all
+
+  tasks:
+  - assert:
+      that: >
+        tpa_2q_repositories is not defined
+        or tpa_2q_repositories == []
+      msg: >-
```
Comment (martinmarques) [inline architectures/lib/commands/check-repositories.yml:13]: We should maybe use `fail_msg` to make it clear that it's a msg on failure of the assert. `msg` is an alias of `fail_msg`

**Pull request merged**